src/status_im/android/core.cljs

```diff
@@ -135,6 +136,7 @@
                       [component])]])]))))})))
 
 (defn init []
+  (dev/dev-setup)
```

Comment:
@flexsurfer, let's move this line to `env.android.main` and `env.ios.main`. I suppose we will use `re-frisk` only for development, so no need to add it to `status-im.*.core` namespaces